scripts/vtk_convert.py

```diff
@@ -0,0 +1,91 @@
+from vtk import vtkPolyDataReader, vtkStructuredPointsReader, \
+    vtkXMLImageDataWriter, vtkXMLPolyDataWriter
+from girder_client import GirderClient
+import os
+import click
+import re
+
+@click.command()
+@click.option('--inputfolderid', required=True, prompt=True, help='Source Folder ID')
+@click.option('--outputfolderid', required=True, prompt=True, help='Destination Folder ID')
+@click.option('--apikey', required=True, prompt=True, help='API Key from Girder')
+@click.option('--hostname', default='https://data.computational-biology.org/api/v1/', prompt=True, help='URL, make sure to include /api/v1/ at the end')
+def main(inputfolderid, outputfolderid, apikey, hostname):
+  gc = GirderClient(apiUrl=hostname)
+  gc.authenticate(apiKey=apikey)
+
+  def getFiles(itemData):
+    print('getting ' + itemData['name'])
+    dataFiles = gc.get('item/' + itemData['_id'] + '/files')
+    return dataFiles[0]
+
+  def getItems(folderData):
+    folderInfo = gc.get('folder/' + folderData['_id'] + '/details')
+    folderItems = gc.get('item',
+      parameters={
+        'folderId': folderData['_id'],
+        'limit': folderInfo['nItems']
+      })
+    return map(getFiles, folderItems)
+
+  gmsFolders = gc.get('folder',
+    parameters={
+      'parentType': 'folder',
+      'parentId': inputfolderid,
+    })
+
+  gmsFileNames = map(getItems, gmsFolders)
+
+  for row in gmsFileNames:
+    for elem in row:
+      oldName = elem['name']
+      print('dowloading ' + oldName)
+      try:
+        gc.downloadFile(elem['_id'], os.path.abspath(oldName))
+
+        reader = vtkPolyDataReader()
+        writer = vtkXMLPolyDataWriter()
+        fileType = ''
```

Comment:
No need to set a default `fileType`, since we're always going to set it inside a conditional. Python variable scoping is function-level, so variables defined within a conditional are available outside it.